### PR TITLE
[#3941]: Introducing wallet / notification not for new user

### LINKED ISSUE #3941: Introducing wallet / notification not for new user
### User Story

As a current user, I want to to have access to signing phrase so that will keep my account safe.

### Description

*Type*: Feature

*Summary*: Mockups here, bottom line: https://zpl.io/agGv1j1

#### Expected behavior
screenshot 2018-04-20 09 33 58

<blockquote><div> Zeplin</div><div><strong><a href="https://zeplin.io/">Zeplin - Login</a></strong></div><div>Collaboration app for UI designers and frontend developers</div></blockquote>

### PR DESCRIPTION
**Pull request by alwx**
Fixes #3941

### CODE REVIEW

**Review by jeluard — APPROVED** [review]

File: src/status_im/ui/screens/wallet/views.cljs [hunk 1]
```diff
@@ -35,6 +34,17 @@
       (:code currency)]]
     [react/text {:style styles/total-value} (i18n/label :t/wallet-total-value)]]])
 
+(defn- backup-seed-phrase []
+  [react/view styles/section
+   [react/touchable-highlight {:on-press #(re-frame/dispatch [:navigate-to :wallet-onboarding-setup])}
```
Comment (flexsurfer) [inline src/status_im/ui/screens/wallet/views.cljs]: it should lead to profile, no?

File: src/status_im/ui/screens/accounts/db.cljs [hunk 2]
```diff
@@ -30,6 +30,7 @@
 (spec/def :account/sharing-usage-data? (spec/nilable boolean?))
 (spec/def :account/dev-mode? (spec/nilable boolean?))
 (spec/def :account/seed-backed-up? (spec/nilable boolean?))
+(spec/def :account/needs-wallet-onboarding? (spec/nilable boolean?))
```
Comment (flexsurfer) [inline src/status_im/ui/screens/accounts/db.cljs]: we have `:seed-backed-up?` for this purpose

**Review by flexsurfer — CHANGES_REQUESTED** [review]

File: src/status_im/ui/screens/wallet/views.cljs (hunk 1 above)
Comment (alwx) [reply src/status_im/ui/screens/wallet/views.cljs]: no

Comment (alwx) [reply src/status_im/ui/screens/wallet/views.cljs]: it should lead to the screen with wallet seed phrase

Comment (flexsurfer) [reply src/status_im/ui/screens/wallet/views.cljs]: https://github.com/status-im/status-react/issues/3941#issuecomment-389893518

File: src/status_im/ui/screens/accounts/db.cljs (hunk 2 above)
Comment (alwx) [reply src/status_im/ui/screens/accounts/db.cljs]: no, it's different. seed phrase you're referring to is different from wallet seed phrase. 
please, check the design first.

Comment (flexsurfer) [reply src/status_im/ui/screens/accounts/db.cljs]: we don't have seed phrase in wallet, it's signing three words

**Review by rasom — APPROVED** [review]

**Comment by alwx** [comment]
Updated. The condition is now different: `as a user if I haven't finished backing up my seed phrase and got some money on the wallet I will get this notification.`

**Comment by rasom** [comment]
apk uploaded to https://i.diawi.com/qRsn7C for branch PR-4319

<blockquote><div><strong><a href="https://i.diawi.com/qRsn7C">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

**Comment by rasom** [comment]
ipa uploaded to https://i.diawi.com/1AEkPD for branch PR-4319

<blockquote><div><strong><a href="https://i.diawi.com/1AEkPD">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

File: src/status_im/ui/components/colors.cljs [hunk 3]
```diff
@@ -7,6 +7,7 @@
 (def white-lighter-transparent "rgba(255, 255, 255, 0.6)") ;; Used for input placeholder color
 (def black "#000000") ;; Used as the default text color
 (def black-transparent "#00000020") ;; Used as background color for rounded button on dark background
+(def black-darker-transparent "#00000033")
```
Comment (flexsurfer) [inline src/status_im/ui/components/colors.cljs]: could you add comment?

File: src/status_im/ui/screens/wallet/views.cljs [hunk 4]
```diff
@@ -92,6 +103,11 @@
                                                   :tint-color :white
                                                   :refreshing false}])}
       [total-section portfolio-value currency]
+      (when (and (not wallet-signing-words-backed-up?)
```
Comment (flexsurfer) [inline src/status_im/ui/screens/wallet/views.cljs]: why `wallet-signing-words` ? it should be `:seed-backed-up?`

File: src/status_im/ui/screens/wallet/views.cljs (hunk 1 above)
Comment (flexsurfer) [reply src/status_im/ui/screens/wallet/views.cljs]: it still should lead to profile

**Review by flexsurfer — CHANGES_REQUESTED** [review]

File: src/status_im/ui/screens/wallet/views.cljs (hunk 1 above)
Comment (alwx) [reply src/status_im/ui/screens/wallet/views.cljs]: will do, but it's super confusing to be honest that these sketches were near onboarding sketches and they are not related to onboarding. and there was nothing about it in the description.

**Comment by rasom** [comment]
apk uploaded to https://i.diawi.com/7rB62B for branch PR-4319

<blockquote><div><strong><a href="https://i.diawi.com/7rB62B">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

**Comment by rasom** [comment]
ipa uploaded to https://i.diawi.com/ffrB4Q for branch PR-4319

<blockquote><div><strong><a href="https://i.diawi.com/ffrB4Q">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

**Comment by rasom** [comment]
apk uploaded to https://i.diawi.com/no9QCQ for branch PR-4319

<blockquote><div><strong><a href="https://i.diawi.com/no9QCQ">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

**Comment by rasom** [comment]
apk uploaded to https://i.diawi.com/jymVwP for branch PR-4319

<blockquote><div><strong><a href="https://i.diawi.com/jymVwP">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

**Comment by rasom** [comment]
ipa uploaded to https://i.diawi.com/1dES2z for branch PR-4319

<blockquote><div><strong><a href="https://i.diawi.com/1dES2z">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

**Comment by rasom** [comment]
ipa uploaded to https://i.diawi.com/BDJky4 for branch PR-4319

<blockquote><div><strong><a href="https://i.diawi.com/BDJky4">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

**Comment by rasom** [comment]
apk uploaded to https://i.diawi.com/akCQM4 for branch PR-4319

<blockquote><div><strong><a href="https://i.diawi.com/akCQM4">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

**Comment by rasom** [comment]
ipa uploaded to https://i.diawi.com/Vsy3kZ for branch PR-4319

<blockquote><div><strong><a href="https://i.diawi.com/Vsy3kZ">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

**Comment by jeluard** [comment]
@flexsurfer Can you take a look again?

**Review by flexsurfer — APPROVED** [review]

**Comment by asemiankevich** [comment]
@alwx when the UI part should be shown? I created account - passed wallet onboarding and dont see this Bacup for seed phrase in Wallet, however the seed phrase i did not back up yet.

**Comment by asemiankevich** [comment]
@alwx nevermind :)

**Pull request merged**